test_data/network_params.yaml

```diff
@@ -20,4 +20,6 @@ assertoor_params:
   run_block_proposal_check: false
   run_blob_transaction_test: true
   tests:
-    - 'https://raw.githubusercontent.com/lambdaclass/lambda_ethereum_rust/refs/heads/main/test_data/el-stability-check.yml'
\ No newline at end of file
+    - 'https://raw.githubusercontent.com/lambdaclass/lambda_ethereum_rust/refs/heads/main/test_data/el-stability-check.yml'
+tx_spammer_params:
+  tx_spammer_extra_args: ["--accounts=10", --txcount=10]
```

Comment:
these parameters are based on our current capabilities?

Reply:
Not really, this is mostly to be able to track the execution instead of having hundreds of transactions (Which by the way we still need to check why we just add one as [shown in the image](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1027#issuecomment-2448479005)). Most of the initial issues regarding the node being unable to propose weren't related to concurrency as I initially thought but to the errors mentioned in the PR description.